apps/services/routers.py

```diff
@@ -262,6 +263,29 @@ async def read_request_details(
     }
 
 
+@services_router.post("/{university_id}/create-students/",
+                      name="create_students_from_file",
+                      response_model=JSENDOutSchema[Union[List[CreateStudentsOut], None]],
+                      summary="Create students from file",
+                      responses={200: {"description": "Successful create students from file response"}},
+                      tags=['Admin dashboard'])
+async def create_students_from_file(
+        request: Request,
+        university_id: int,
+        file: UploadFile = File(),
+        user=Depends(get_current_user),
+        session: AsyncSession = Depends(get_async_session)):
+    response = await service_handler.create_students_from_file(
```

Comment:
again  `create_students_from_file` and similar phrases  to `create_student_list_from_file `

Reply:
ok